packages/core/src/utils/vercelWaitUntil.ts

```diff
@@ -18,8 +18,7 @@ export function vercelWaitUntil(task: Promise<unknown>): void {
     // @ts-expect-error This is not typed
     GLOBAL_OBJ[Symbol.for('@vercel/request-context')];
 
-  const ctx =
-    vercelRequestContextGlobal?.get && vercelRequestContextGlobal.get() ? vercelRequestContextGlobal.get() : {};
+  const ctx = vercelRequestContextGlobal?.get?.();
```

Comment:
is this identical or should it rather be
```suggestion
  const ctx = vercelRequestContextGlobal?.get?.() ?? {}
```
?